homeassistant/components/sensor/zwave.py

```diff
@@ -33,26 +34,36 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
     # if 1 in groups and (NETWORK.controller.node_id not in
     #                     groups[1].associations):
     #     node.groups[1].add_association(NETWORK.controller.node_id)
+    name = '{}.{}'.format(DOMAIN, zwave.object_id(value))
+    node_config = customize.get_overrides(hass, zwave.DOMAIN, name)
```

Comment:
Good point.

So either enable force_update for zwave sensors with polling_intensity (current state of this PR) or for all zwave sensors regardless of polling_intensity.

Now when I'm thinking about it, polling_intensity is only needed for non-perfect sensors that can't be configured to reports at required intervals on their own. But even "perfect" sensors reports are "sup to date".

So I propose to mark all zwave sensors as force_update.

WDYT?